sql/hive/src/test/scala/org/apache/spark/sql/sources/HadoopFsRelationTest.scala

```diff
@@ -760,23 +775,27 @@ abstract class HadoopFsRelationTest extends QueryTest with SQLTestUtils with Tes
   // requirement.  We probably want to move this test case to spark-integration-tests or spark-perf
   // later.
   test("SPARK-8406: Avoids name collision while writing files") {
```

Comment:
Just move this to ParquetHadoopFsRelationSuite.scala

Reply:
+1